slides.md

```diff
@@ -332,3 +332,36 @@ To alter the state of a component, you need to invoke `this.setState` inside tha
 The second case is required when your update depends on the current state. The function receives the current state and must return the new state (which is then merged, like the previous case). Since state updates are asynchronous, this is necessary because you're not ensured to have the latest state inside `this.state`.
 
 Add a callback to the various components, and update the state inside `App`.
+
+# Promises
+
+The standard interface to do asynchronous work in JS is `Promise`. A promise is an object representing something that will end in the future.
```

Comment:
Could you maybe give an example of "something"? Maybe a reference from the previous parts?

Reply:
as in: "for example, an network request"?

Reply:
2b9bbfa